manifests/integrations/network.pp

```diff
@@ -20,10 +20,25 @@
 #
 #
 class datadog_agent::integrations::network(
-  Boolean $collect_connection_state = false,
-  Array[String] $excluded_interfaces = ['lo','lo0'],
-  String $excluded_interface_re = '',
-  Boolean $combine_connection_states = true,
+  Boolean $collect_connection_state          = false,
+  Boolean $collect_connection_queues         = false,
+  Array[String] $excluded_interfaces         = ['lo','lo0'],
+  String $excluded_interface_re              = '',
+  Boolean $combine_connection_states         = true,
+  Boolean $collect_rate_metrics              = true,
+  Boolean $collect_count_metrics             = false,
+  String $conntrack_path                     = '/usr/sbin/conntrack',
+  Boolean $use_sudo_conntrack                = true,
+  Array[String] $whitelist_conntrack_metrics = [],
+  Array[String] $blacklist_conntrack_metrics = [],
+  Boolean $collect_aws_ena_metrics           = false,
+  Boolean $collect_ethtool_metrics           = false,
```

Comment:
Hmm, I don't see this configuration value anywhere in the sample config or the code of the integration. Where did you find it?